entities/src/main/java/com/flipkart/varadhi/entities/TopicResource.java

```diff
@@ -31,7 +31,7 @@ public TopicResource(
 
     @Override
     public String uniqueKeyPath() {
-        return String.format("/%s/%s/%s", RESOURCE_TYPE_NAME, project, getName());
+        return String.format("%s/%s/%s", RESOURCE_TYPE_NAME, project, getName());
```

Comment:
@yagami-kunal e2e test is little fragile as of now. You can change the test to use uniqueKeyPath() in its duplicate topic assertion.